consultation_analyser/support_console/views/consultations.py

```diff
@@ -12,7 +12,13 @@
 def index(request: HttpRequest) -> HttpResponse:
     if request.POST:
         try:
-            dummy_data.DummyConsultation(include_themes=False, number_questions=2)
+            dummy_data.DummyConsultation(include_themes=False, number_questions=10)
```

Comment:
Why not return the generated consultation from here?

Reply:
I wonder if `DummyConsultation` ought to just be a function, given right now it's nothing but a constructor with side effects

Reply:
I think it is easier as a function - have refactored.

Reply:
Great